web/src/components/molecules/Asset/Asset/AssetBody/unzipFileList.tsx

```diff
@@ -0,0 +1,96 @@
+import { CaretDownFilled } from "@ant-design/icons";
```

Comment:
Please make sure to import this into icons.ts and use the Icon component here

Reply:
Icon component only support "~Outlined" icons so far. I think that we should add "~Filled" and "~TwoTone" to the Icon component.

Reply:
I used "CaretDownOutlined" instead of "CaretDownFilled" for now.